mappings/net/minecraft/block/BlockSoundGroup.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_475 net/minecraft/block/BlockSoundGroup
```

Comment:
Supposed to be `net/minecraft/sound/BlockSoundGroup`, according to https://github.com/FabricMC/yarn/blob/23w35a/mappings/net/minecraft/sound/BlockSoundGroup.mapping